src/test/java/uk/gov/ida/matchingserviceadapter/services/EidasMatchingServiceTest.java

```diff
@@ -0,0 +1,67 @@
+package uk.gov.ida.matchingserviceadapter.services;
+
+import org.beanplanet.messages.domain.Message;
```

Comment:
Don't worry: all references to the Beanplanet library HAVE BEEN REMOVED in the next commit changeset!